### DISCUSSION TITLE
Adding a new M2O relation to a data model removes already existing O2M

### DISCUSSION THREAD

**Discussion by sagerio** [directus/directus#7009] [Archive]
As the title says, I'm trying to add a new M2O relation (simple "created by" to a user data model") via "Create field -> M2O relation".

This works, but immediately after the already existing O2M relation in this data model is cleared.

I've no glue if I set something wrong or there must be used another way to add such a field to a model with already existing relations?

Regards,
Robert

**Comment by rijkvanzanten** [comment]
Mind [opening a new Issue](https://github.com/directus/directus/issues/new?assignees=&labels=Bug+%28Potential%29&template=bug_report.yml)?

**Reply by sagerio** [reply]
I will, thanks!
